lib/server.js

```diff
@@ -613,12 +606,26 @@ Server.prototype.start = function start(cb) {
   ], done);
 
   function appListen(callback) {
-    debug('app listen on %d', self._listenPort);
+    debug('Initializing http listen on port %d', self._listenPort);
     try {
-      self._app.listen(self._listenPort, function(err) {
+      http.createServer(self._baseApp).listen(self._listenPort, function(err) {
         if (err) return callback(err);
+
+        var address = this.address();
+        if (address.port) {
+          console.log('%s: StrongLoop PM listening on port `%s`',
+            self._originalCommand,
+            address.port);
+        } else {
```

Comment:
This listen handler appears to be written to handle both the app and the control channel.. but they are completely separate.